ContractsApi/V1/Domain/Charges.cs

```diff
@@ -9,5 +9,6 @@ public class Charges
         public string SubType { get; set; }
         public Frequency Frequency { get; set; }
         public decimal? Amount { get; set; }
+        public string VatRate { get; set; }
```

Comment:
More a question than a change to the code - where do we normally document these fields? I wonder if there's somewhere were we could mention that the VatRate is a code rather than an amount?

Reply:
I think SwaggerHub...? It doesn't look like there's an automated publishing process in the pipeline but I can definitely add it manually